templates/common/infra/bicep/core/host/aks-managed-cluster.bicep

```diff
@@ -12,7 +12,7 @@ param location string = resourceGroup().location
 param tags object = {}
 
 @description('Kubernetes Version')
-param kubernetesVersion string = '1.27.7'
+param kubernetesVersion string = '1.28.9'
```

Comment:
Just wondering - I remember a few weeks ago running into some issue where there patch version we were using had been removed.  Is it better to just use `1.28` here?  I think it will select the latest patch version then?

Reply:
Makes sense.  In early testing I was seeing issues with some minor version differences.  In new tests it is working fine with just the major and minor versions.